packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -75,11 +75,12 @@ function InterfaceSkeleton(
 	const mergedLabels = { ...defaultLabels, ...labels };
 
 	const headerVariants = {
-		hidden: isDistractionFree ? { opacity: 0 } : { opacity: 1 },
+		hidden: { opacity: 0 },
```

Comment:
@draganescu I think the variants should be static 'states', and framer motion can then switch between those states like a state machine. Because of that I've removed the ternary and instead added a new state (`distractionFreeInactive`).

I noticed similar code in a few other places and it might be a good idea to update those too.

Reply:
This is awesome @talldan - it did not occur to me to try this at all 😢  and I reached that cumbersome duplication because I couldn't get framer motion to not animate when distraction free was off (on toggling it one was left with a sort of "left over" effect) - hence the rerendering hence the focus bug.